### PR TITLE
Permit named_params with suffix

### PR DESCRIPTION
**Pull request by tanguyantoine**
With rails we can have this kind of routes :

/:named_param-suffix-for-params

so this commit permit to translate suffix-for-params

### CODE REVIEW

**Comment by tagliala** [comment]
Hi, we need proper tests for this feature

**Comment by tanguyantoine** [comment]
Updated

**Comment by tanguyantoine** [comment]
up

**Comment by tagliala** [comment]
@tanguyantoine please rebase this one once again

**Comment by tagliala** [comment]
@tanguyantoine specs are failing. Please note that the last merged PR interested the portion of code you are editing

File: lib/route_translator/translator.rb [hunk 1]
```diff
@@ -121,7 +121,10 @@ def self.default_locale?(locale)
     # segment is blank, begins with a ":" (param key) or "*" (wildcard),
     # the segment is returned untouched
     def self.translate_path_segment(segment, locale)
-      return segment if segment.blank? or segment.include?(":") or segment.starts_with?("(") or segment.starts_with?("*")
+      return segment if segment.blank?
+      named_param, hyphenized = segment.split('-'.freeze, 2) if segment.starts_with?(":".freeze)
+      return "#{named_param}-#{translate_path_segment(hyphenized.dup, locale)}" if hyphenized
+      return segment if segment.starts_with?("(".freeze) or segment.starts_with?("*".freeze) or segment.starts_with?(":".freeze)
```
Comment (tagliala) [inline lib/route_translator/translator.rb]: This line changed via #118 

https://github.com/enriclluelles/route_translator/pull/118/files#diff-6500de086a3ee34b17d3ed33ff3af908L124

**Comment by tagliala** [comment]
Failure seems unrelated. Maybe I've messed up with something

**Comment by tanguyantoine** [comment]
Yep

**Comment by tagliala** [comment]
Changed a lot of things, cherry-picked your commit here:

https://github.com/enriclluelles/route_translator/tree/permit-named-params

Everything seems fine. Could you please give a try to the above branch?

**Comment by tanguyantoine** [comment]
What should i have to do?

**Comment by tagliala** [comment]
check if that branch is working fine for your use case

**Comment by tanguyantoine** [comment]
If tests are passing, it's good for me: https://github.com/enriclluelles/route_translator/blob/permit-named-params/test/integration/generated_path_test.rb#L57

**Comment by tagliala** [comment]
:+1: merged to develop branch, thanks!

**Pull request closed without merging**